shared/gh/files/config.json

```diff
@@ -0,0 +1,22 @@
+{
+    "terms": [
```

Comment:
This would work nicely for the current year, but we'll probably need something that can handle multiple years? e.g. for archiving or creating the next year before the current one finished.